rest/route/service.go

```diff
@@ -168,6 +168,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/tasks/{task_id}").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeGetTaskRoute(sc))
 	app.AddRoute("/tasks/{task_id}").Version(2).Patch().Wrap(checkUser, addProject, editTasks).RouteHandler(makeModifyTaskRoute(sc))
 	app.AddRoute("/task/{task_id}/annotation").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeFetchAnnotationsByTask(sc))
+	app.AddRoute("/task/{task_id}/annotation").Version(2).Put().Wrap(checkUser, viewTasks, editTasks).RouteHandler(makePutAnnotationsByTask(sc))
```

Comment:
nit: this should be tasks (sorry 170 should also be tasks, I didn't catch that before). Also, editTasks is already a higher permission level than viewTasks so you don't need both (I know we're switching this middleware but that principle will still apply)